app/waf/src/models/action.model.ts

```diff
@@ -0,0 +1,77 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Action extends Entity {
```

Comment:
You may inherits CommonEntity who is in models/common.model.ts. That class helps you to define id, name, createdAt and updatedAt.

Reply:
will correct it.